flink-architecture-tests/pom.xml

```diff
@@ -177,7 +177,7 @@ under the License.
 			<dependency>
 				<groupId>org.apache.flink</groupId>
 				<artifactId>flink-connector-kafka</artifactId>
-				<version>${project.version}</version>
+				<version>3.0.0-1.17</version>
```

Comment:
This dependency should be removed, with the connector defining it's own prod/test architecture tests (similar to what we did for other connectors).